fs/dafilesrv/dafilesrv.cpp

```diff
@@ -590,6 +590,15 @@ int main(int argc, const char* argv[])
             if (componentExpert)
                 synchronizePTree(expert, componentExpert, false, true);
 
+            // merge in bare-metal dafilesrv component cert settings into newConfig
+            IPropertyTree *componentCert = nullptr;
```

Comment:
trivial: could combine with the next line (I wouldn't have commented if not only comment).